### DISCUSSION TITLE
Technical Report (the missing manual/review)

### DISCUSSION THREAD

**Discussion by niksedk** [SubtitleEdit/subtitleedit#5338] [General]
Pietro Moras has provided a independent 150 page technical report describing the main program window and all menu/context-menu elements.

The PDF is available here: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.4/SE-TR.211200.pdf

Comments are welcome :)

**Comment by Mishasama** [comment]
404

**Reply by niksedk** [reply]
thx - link fixed :)

**Comment by Blek182** [comment]
Hi, i am using portable mode. The Whisper function not working. There is no model to download. Download model not working. Tiny, Small, Medium, Large All not working when i click Download nothing happpen. And then i tried download manual. And put in the folder model. The format is .pt rite? and not showing in Subtitle Edit. Re open the software, same result still not showing the model. Please check. Thank you.

*Someone face same problem too, cannot download model whisper. And already posted the issue there: https://www.nikse.dk/subtitleedit/techreport

**Comment by niksedk** [comment]
The models urls have changed... read more here: https://github.com/SubtitleEdit/subtitleedit/issues/7158
